server/c2/http.go

```diff
@@ -810,7 +810,14 @@ func newHTTPSessionID() string {
 func getRemoteAddr(req *http.Request) string {
 	ipAddress := req.Header.Get("X-Real-Ip")
 	if ipAddress == "" {
-		ipAddress = req.Header.Get("X-Forwarded-For")
+		xForwardedFor := req.Header.Get("X-Forwarded-For")
+		if xForwardedFor != "" {
+			ips := strings.Split(xForwardedFor, ",")
+			if len(ips) > 0 {
+				// Extracts original client ip address
+				ipAddress = strings.TrimSpace(ips[0])
+			}
```

Comment:
Looks like we're missing the `else` case here (the old behavior of only one IP address in the header)

Reply:
good point, I'll push an update